core/src/apps/nostr/sign_event.py

```diff
@@ -0,0 +1,35 @@
+from typing import TYPE_CHECKING
+
+from apps.common.keychain import auto_keychain
+
+if TYPE_CHECKING:
+    from trezor.messages import NostrEventSignature, NostrSignEvent
+
+    from apps.common.keychain import Keychain
+
+
+@auto_keychain(__name__)
+async def sign_event(msg: NostrSignEvent, keychain: Keychain) -> NostrEventSignature:
+    from ubinascii import hexlify
+
+    from trezor.crypto.curve import secp256k1
+    from trezor.crypto.hashlib import sha256
+    from trezor.messages import NostrEventSignature
+
+    address_n = msg.address_n
+    created_at = msg.created_at
+    kind = msg.kind
+    tags = msg.tags
+    content = msg.content
+
+    node = keychain.derive(address_n)
+    pk = node.public_key()[-32:]
```

Comment:
Can we do this? Don't we care about the public key parity?

Reply:
> Can we do this? Don't we care about the public key parity? Or does nostr use x-only keys?

From what I understand, Nostr uses X-only keys, indeed.